app/views/games/show.html.erb

```diff
@@ -1,22 +1,18 @@
-<div id='chessboard'>
+<div class="col-xs-12 col-sm-8 col-sm-offset-2 col-md-6 col-md-offset-3 no-padding">
 	<%= hidden_field :game_id, '', name: nil, id: nil, data: { game_id: @game.id } %>
```

Comment:
My apologies. I wasn't being clear. What I meant was after I made the changes the 'chessboard' id was no longer defined. There was fixed height and width using the media queries, margin and box-shadow. The height, width and margin were not needed after implementing Bootstrap's column classes. The box-shadow I can add back in if you'd like but I thought it would be kind of a waste once the board goes full screen width on small screens. Shadows can look funny if they're cutoff in anyway. After these changes there was nothing left to define the chessboard.

I can definitely appreciate your stance on labels. It's a good standard to keep so I'll try to use them as much as possible. Are you open to using comments as labels or just css selectors?

As I mentioned before, the 'no-padding' class isn't intended to be used exclusively for the chessboard. Many of Bootstrap's classes add padding to the left and right sides. There are instances were this padding isn't needed. I definitely could just define the attribute in a given element's class but this won't address the elements that don't have their own special class or id. I know in this PR it isn't evidenced at all that 'no-padding' will be used with other elements that don't have a special class association but I definitely intend to use this in other areas of the app simply because I know Bootstrap will be used. If there is a simpler solution that will address those elements that don't have their own special class then I'd be happy to adjust the code.

I can add the chessboard id back in or add a comment to use as a label for the board. What do you think is the best approach here? I'd be happy to make any changes you see fit.